Crystal.sublime-build

```diff
@@ -2,8 +2,7 @@
 	"cmd": ["crystal", "run", "--no-color", "$file"],
 	"file_regex": "^Error in (?<filename>.+\\.cr):(?<line_number>[0-9]*):?(?<column_number>[0-9]*): (?<message>.+)$",
 	"selector": "source.crystal",
-	"quiet": true,
```

Comment:
Please leave it as is for now. It should be another issue to discuss whether to supress the output.

Reply:
sorry I forgot to put that on